Testing/Data/SystemTest/ca4903d7d916d5fb426eafa165a78062

```diff
@@ -0,0 +1,1550 @@
+#Title: diamond cw0.533 4.22e12 60Hz [10x30]
+#Inst: POWGEN.prm
+#Binning: ddperp   0.0130986    0.1000000
+#Bank: 1
+#2theta   lambda   d-value   dp-value   counts
```

Comment:
If these files have been uploaded to the system data store we can remove them from the repo, as the system test data target will fetch them.